llvm/lib/Target/Sparc/SparcAsmPrinter.cpp

```diff
@@ -317,10 +318,11 @@ void SparcAsmPrinter::printOperand(const MachineInstr *MI, int opNum,
 #ifndef NDEBUG
```

Comment:
The entire thing undef this `#ifndef` is dead code.
This method is only called when printing inline asm.

Reply:
Yes, I think it should be removed.